File: Sorting/bubbleSort.go
package sorting

func BubbleSort(array []int) {
    for i := 0; i < len(array); i++ {
        for j := i; j < len(array); j++ {
            if array[i] > array[j] {
                array[i], array[j] = array[j], array[i]
            }
        }
    }
}

File: DSA/Sorting/bubble_sort.py
from helpers import get_test_case, swap


def bubble_sort(array: "list[int]"):
    for i in range(len(array)):
        for j in range(i, len(array)):
            if array[j] < array[i]:
                swap(array, i, j)


a = get_test_case()

print(a)

bubble_sort(a)

print(a, a == sorted(a))
